src/Explorer/Panes/SettingsPaneR/index.tsx

```diff
@@ -0,0 +1,433 @@
+
+    setIsExecuting(false);
+    NotificationConsoleUtils.logConsoleInfo(
+      `Updated items per page setting to ${LocalStorageUtility.getEntryNumber(StorageKey.ActualItemPerPage)}`
+    );
+    NotificationConsoleUtils.logConsoleInfo(
+      `${crossPartitionQueryEnabled ? "Enabled" : "Disabled"} cross-partition query feed option`
+    );
+    NotificationConsoleUtils.logConsoleInfo(
+      `Updated the max degree of parallelism query feed option to ${LocalStorageUtility.getEntryNumber(
+        StorageKey.MaxDegreeOfParellism
+      )}`
+    );
+
+    if (shouldShowGraphAutoVizOption) {
+      NotificationConsoleUtils.logConsoleInfo(
+        `Graph result will be displayed as ${
+          LocalStorageUtility.getEntryBoolean(StorageKey.IsGraphAutoVizDisabled) ? "JSON" : "Graph"
+        }`
+      );
+    }
+
+    NotificationConsoleUtils.logConsoleInfo(
+      `Updated query setting to ${LocalStorageUtility.getEntryString(StorageKey.SetPartitionKeyUndefined)}`
+    );
+
+    close();
+  };
+
+  const isCustomPageOptionSelected = () => {
+    return pageOption === Constants.Queries.CustomPageOption;
+  };
+
+  const isUnlimitedPageOptionSelected = () => {
+    return pageOption === Constants.Queries.UnlimitedPageOption;
+  };
+
+  const onUnlimitedPageOptionKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.UnlimitedPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onCustomPageOptionsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setPageOption(Constants.Queries.CustomPageOption);
+      event.stopPropagation();
+    }
+  };
+
+  const onJsonDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("true");
+      event.stopPropagation();
+    }
+  };
+
+  const onGraphDisplayResultsKeyDown = (event: KeyboardEvent) => {
+    if (event.keyCode === Constants.KeyCodes.Enter || event.keyCode === Constants.KeyCodes.Space) {
+      setGraphAutoVizDisabled("false");
+      event.stopPropagation();
+    }
+  };
+
+  const _loadSettings = () => {
+    setIsExecuting(true);
+    try {
+      setPageOption(
+        LocalStorageUtility.getEntryNumber(StorageKey.ActualItemPerPage) === Constants.Queries.unlimitedItemsPerPage
+          ? Constants.Queries.UnlimitedPageOption
+          : Constants.Queries.CustomPageOption
+      );
+      setCustomItemPerPage(LocalStorageUtility.getEntryNumber(StorageKey.CustomItemPerPage));
+    } catch (exception) {
+      setFormErrors("Unable to load your settings");
+      setFormErrorsDetails(exception);
+    } finally {
+      setIsExecuting(false);
+    }
+  };
+
+  return (
+    <div data-bind="visible: visible, event: { keydown: onPaneKeyDown }">
```

Comment:
Remove any `data-bind` attributes. They are for knockout and not needed

Reply:
done